fedgov-cv-ISDcreator-webapp/src/main/webapp/js/depositor.js

```diff
@@ -403,7 +403,7 @@ function createMessageJSON()
                 "referenceLat": feature.attributes.LonLat.lat,
                 "referenceLon": feature.attributes.LonLat.lon,
                 "referenceElevation": feature.attributes.elevation,
-                "roadAuthorityId": feature.attributes.roadAuthorityId,
+                "roadAuthorityId": feature.attributes.roadAuthorityId?.split("."),

```

Comment:
Currently this field is being passed as array of strings

  "roadAuthorityId": [
          "12",
          "21",
          "12"
        ],

Please update it to pass as array of integers:

  "roadAuthorityId": [
          12,
          21,
          12
        ],